src/states/elder/mod.rs

```diff
@@ -2157,7 +2179,7 @@ impl Elder {
     }
 
     pub fn set_next_relocation_interval(&mut self, interval: Option<(XorName, XorName)>) {
```

Comment:
Why not change this method to take `XorTargetInterval` too?

Reply:
Same reason, really, I did not really want to change all the test code, but if you think it is needed, I add an extra commit (or same commit as other suggestion).

Reply:
I wouldn't say it's needed, but it would make the code cleaner. Currently there are two different representations of xor-name intervals for no good reason which adds complexity and confusion.